src/Generator.Rewrite/Program.cs

```diff
@@ -255,7 +248,7 @@ void RemoveSupportingAttributes(TypeDefinition type)
 
         // Create body for method
         static void ProcessMethod(MethodDefinition wrapper, MethodDefinition native, int slot,
-                                  FieldDefinition entry_points, IEnumerable<string> options)
+                                  FieldDefinition entry_points, List<string> options)
```

Comment:
Follow the "IEnumerable ToList internally" pattern again here if multiple enumeration is required.